docs/reference/migration/migrate_5_0/java.asciidoc

```diff
@@ -225,3 +225,40 @@ The `addSuggestion` method now required the user specified suggestion name, prev
 ===== SuggestionBuilder
 
 The `field` setter has been deleted. Instead the field name needs to be specified as constructor argument.
+
+==== SearchSourceBuilder
+
+All methods which take an `XContentBuilder`, `BytesReference` `Map<String, Object>` or `bytes[]` have been removed in favor of providing the
+relevant builder object for that feature. This means that all search requests can now be validated at call time which results in much clearer
```

Comment:
can you add an example of what we mean by builder object? Also do we still want to remove the builder suffix? In that case we may want to start here by using a different name?